frontend/src/i18n/locales/de.yaml

```
@@ -18,7 +18,14 @@ general:
   leave-page-confirmation: >
     Seite verlassen? Vorgenommene Änderungen sind möglicherweise noch nicht gespeichert!
   logo-alt: Das Logo von „{{title}}“
+  no-root-children: Noch keine Seiten...
 
+welcome:
+  title: Willkommen zu Tobira!
+  body: >
+    Die Hauptseite scheint noch leer zu sein! Um Inhalte oder weitere Seiten
+    hinzuzufügen, melden Sie sich zuerst mit einem Account (mit Moderator-Rechten)
+    an. Danach finden Sie auf der linken Seite Buttons zum Editieren von Seiten.
```

Not 100% sure but maybe

```suggestion
    hinzuzufügen, melden Sie sich zuerst mit einem Konto (mit Moderator-Rechten)
    an. Danach finden Sie auf der linken Seite Schaltflächen zum Editieren von Seiten.
```

Or something like that? :thinking: Could also make it `Zugang` and `Bedienelemente` ...